### PR TITLE
Make docs

### PR DESCRIPTION
**Pull request by e-koch**
First pass through basic documentation.

### CODE REVIEW

File: docs/feathering_cube.rst [hunk 1]
```diff
@@ -0,0 +1,16 @@
+.. _feathercubes:
+
+Feathering two cubes
+====================
+
+.. note:: In progress...
+
+.. warning:: The method desribed here is currently not tested. Check your results carefully when using.
```
Comment (keflavich) [inline docs/feathering_cube.rst:8]:
```suggestion
.. warning:: The method desribed here is currently not tested as part of the continuous integration suite. Check your results carefully when using.
```

File: docs/feathering_cube.rst [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+Two cubes can currently be combined with the `~uvcombine.fourier_combine_cubes`.
+This method is appropriate for two cube, each with a single beam size (i.e., the beam
```
Comment (keflavich) [inline docs/feathering_cube.rst:11]:
```suggestion
This method is appropriate for two cubes that each have a single beam size (i.e., the beam
```

File: docs/feathering_cube.rst [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+does not change between spectal channels). The low resolution beam size is also set
```
Comment (keflavich) [inline docs/feathering_cube.rst:12]:
```suggestion
does not change between spectral channels). The low resolution beam size is also set
```

File: docs/feathering_images.rst [hunk 4]
```diff
@@ -0,0 +1,36 @@
+.. _featherimages:
+
+Feathering two images
+=====================
+
+`~uvcombine.feather_simple` is the primary function for feathering.
+
+Images can be passed as a FITS filename (as a string), FITS HDU, or a `~spectral_cube.Projection`.
+The latter is recommended to allow close check of the beam sizes passed for the high- and
```
Comment (keflavich) [inline docs/feathering_images.rst:9]:
```suggestion
The latter is recommended to allow for a close check of the beam sizes passed for the high- and
```

File: docs/install.rst [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+* `six <http://pypi.python.org/pypi/six/>`__
+
+Installation
+------------
+
+..
+    To install the latest stable release, you can type::
+
+..
+    pip install uvcombine
```
Comment (keflavich) [inline docs/install.rst:21]: do we not have a pypi version going?

Comment (e-koch) [reply docs/install.rst:21]: Not yet.

Comment (e-koch) [reply docs/install.rst:21]: Should wait until we finish the tests, IMO.

**Comment by e-koch** [comment]
I enabled PR builds on RTD, but it doesn't seem to have switched on yet.

@keflavich  From my side, this can be merged if there are no more comments.

**Pull request merged**